Automattic/simplenote-macos
# Privacy Policy: Enhanced UI

### Details:
In this PR we're enhancing the Privacy UI with extra details / external links.

cc @bummytime or @aerych
Gentlemen, may i bug you with a super quick PR?

**THANK YOU!!!**
Closes #376

### Testing:
1. Log into your account
2. Open `System Bar` > `Simplenote` > `Privacy Settings`
3. Verify the UI makes sense
4. Verify that clicking over all of the textFields result in either the Privacy URL or Cookies URL onscreen.

---

<img width="460" alt="Screen Shot 2019-08-21 at 11 49 20 AM" src="https://user-images.githubusercontent.com/1195260/63442572-ba3fbd80-c409-11e9-885f-a3edf99e94ac.png">
